quadcopter_brain/src/quadcopter_brain/quadcopter_brain.py

```diff
@@ -131,3 +131,23 @@ def land_on_fiducial_simple(self):
                      'altitude': 1.0}
             self.go_to_waypoints([waypt])
         self.land()
+
+    def land_on_fiducial_incremental(self):
+        found, _, _ = self.find_landing_site()
+        alt = -1.0
+        if found:
+            alt = self.quadcopter.current_rel_alt
+            while alt > 4.0:
+                goal_lat, goal_long, goal_vertical_dist = \
+                    self.landing_site.get_average_lat_long(self.quadcopter)
+                if goal_lat is not None:
```

Comment:
I think this would be better to be goal_lat != None since it's checking the value.